dev-python/logfury/logfury-0.1.2-r1.ebuild

```diff
@@ -3,13 +3,14 @@
 
 EAPI=6
 
-PYTHON_COMPAT=( python3_6 )
+PYTHON_COMPAT=( python3_6 python3_7 python3_8 )
+DISTUTILS_USE_SETUPTOOLS=bdepend
 
 inherit distutils-r1
 
 DESCRIPTION="Boilerplate library for logging method calls"
 HOMEPAGE="https://github.com/ppolewicz/logfury https://pypi.org/project/logfury/"
-SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+SRC_URI="https://github.com/ppolewicz/${PN}/archive/${PV}.tar.gz -> ${P}a.tar.gz"
```

Comment:
Use `.gh.tar.gz` not to introduce false versions.

Reply:
Now uses .gh instead of .a